Import ruby WPTs

anttijk opened the pull request:
#### 287b8954df801df63345e6f2c3dd87845fa6f91e
<pre>
Import ruby WPTs
<a href="https://bugs.webkit.org/show_bug.cgi?id=253826">https://bugs.webkit.org/show_bug.cgi?id=253826</a>
rdar://106642220

Reviewed by Tim Nguyen.

Import web-platform-tests/css/css-ruby.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base-container-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base-container.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-container-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-container.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-000-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-000.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-base-container-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-base-container-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-base-container.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-abs-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-abs.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-float-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-float.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/improperly-contained-annotation-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/improperly-contained-annotation-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/inheritance-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/inheritance.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/intra-base-white-space-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/intra-base-white-space-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/line-break-around-ruby-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/line-break-around-ruby-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/line-spacing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/line-spacing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/nested-ruby-pairing-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/nested-ruby-pairing-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/nested-ruby-pairing-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-align-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-align-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-align-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-align-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-merge-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-merge-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-merge-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-merge-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-position-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-position-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-position-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-position-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rb-display-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rb-display-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rbc-rtc-basic-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rbc-rtc-basic-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-block-ruby-expected.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-block-ruby.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-ruby-expected.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-ruby-ref.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-ruby.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rt-display-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rt-display-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-001.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001a-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001a.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002a-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002a.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-annotation-pairing-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-annotation-pairing-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-container-abs-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-container-abs.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-container-float-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-container-float.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-different-size-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-different-size-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-different-size.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-model-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-model-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-model-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-float-handling-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-float-handling-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-float-handling-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-lang-specific-style-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-lang-specific-style-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-lang-specific-style-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-layout-internal-boxes-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-layout-internal-boxes-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-layout-internal-boxes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-no-transform-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-no-transform-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-no-transform.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-position-alternate-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-position-alternate.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-position-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-position.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-noruby.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-opaqueruby-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-opaqueruby.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-transparentruby-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-transparentruby.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-span-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-span-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-span-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-collapse-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-collapse-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-collapse.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001a-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001a.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001b-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001b.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002a-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002a.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002b-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002b.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/support/ruby-common.css: Added.
(rbc):
([pseudo]):
([pseudo] &gt; rt):
([pseudo] &gt; rt[pseudo]):
(.large):
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/support/ruby-dynamic-insertion.js: Added.
(window.onload):
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/support/ruby-dynamic-removal.js: Added.
(getElements):
(window.onload):
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/support/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/w3c-import.log: Added.

Canonical link: <a href="https://commits.webkit.org/261621@main">https://commits.webkit.org/261621@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ccb7c2e847dc1f88c506c924b3b904db158836cf

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/112209 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/21344 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/90/builds/836 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/4010 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/120854 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/22691 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/12537 "Built successfully") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/86/builds/4771 "107 failures") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/117972 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/16859 "3 failures") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/100033 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/105271 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/98816 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/88/builds/574 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/45865 "2 flakes 126 failures 1 missing results") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/13751 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/89/builds/614 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/93856 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/14429 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/84/builds/10011 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/19794 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/52613 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/8098 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/16223 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Looks good, but please link to the wpt revision in the commit message.

anttijk committed:
Import ruby WPTs
https://bugs.webkit.org/show_bug.cgi?id=253826
rdar://106642220

Reviewed by Tim Nguyen.

Import web-platform-tests/css/css-ruby.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base-container-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base-container.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-base.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-container-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/abs-in-ruby-container.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/block-ruby-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-000-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-000.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/br-clear-all-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-base-container-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-base-container-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-base-container.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-abs-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-abs.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-float-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-float.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/empty-ruby-text-container.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/improperly-contained-annotation-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/improperly-contained-annotation-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/inheritance-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/inheritance.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/intra-base-white-space-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/intra-base-white-space-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/line-break-around-ruby-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/line-break-around-ruby-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/line-spacing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/line-spacing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/nested-ruby-pairing-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/nested-ruby-pairing-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/nested-ruby-pairing-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-align-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-align-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-align-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-align-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-merge-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-merge-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-merge-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-merge-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-position-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-position-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-position-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/ruby-position-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/parsing/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rb-display-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rb-display-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rbc-rtc-basic-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rbc-rtc-basic-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-block-ruby-expected.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-block-ruby.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-ruby-expected.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-ruby-ref.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/root-ruby.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rt-display-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/rt-display-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-001.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001a-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-001a.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002a-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-align-002a.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-annotation-pairing-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-annotation-pairing-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-autohide-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-container-abs-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-container-abs.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-container-float-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-container-float.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-different-size-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-different-size-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-base-different-size.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-bidi-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-generation-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-model-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-model-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-box-model-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-insertion-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-dynamic-removal-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-float-handling-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-float-handling-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-float-handling-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-inlinize-blocks-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intra-level-whitespace-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-intrinsic-isize-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-justification-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-lang-specific-style-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-lang-specific-style-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-lang-specific-style-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-layout-internal-boxes-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-layout-internal-boxes-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-layout-internal-boxes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-break-suppression-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-line-breaking-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-no-transform-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-no-transform-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-no-transform.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-position-alternate-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-position-alternate.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-position-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-position.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-noruby.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-opaqueruby-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-opaqueruby.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-transparentruby-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-reflow-001-transparentruby.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-span-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-span-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-span-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-collapse-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-collapse-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-collapse.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001a-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001a.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001b-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-001b.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002a-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002a.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002b-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-text-combine-upright-002b.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-whitespace-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/ruby-with-floats-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/support/ruby-common.css: Added.
(rbc):
([pseudo]):
([pseudo] > rt):
([pseudo] > rt[pseudo]):
(.large):
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/support/ruby-dynamic-insertion.js: Added.
(window.onload):
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/support/ruby-dynamic-removal.js: Added.
(getElements):
(window.onload):
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/support/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ruby/w3c-import.log: Added.

Canonical link: https://commits.webkit.org/261621@main
